docs/src/ode_fitting/prediction_error_method.md

```diff
@@ -1,4 +1,5 @@
 # Prediction error method (PEM)
```

Comment:
```suggestion
# [Prediction error method (PEM)](@id pemethod)
```

Reply:
https://github.com/SciML/ModelingToolkit.jl/blob/master/docs/src/basics/Composition.md#:~:text=Building%20Reusable%20Components%5D-,(%40id%20components),-The%20symbolic%20models